pkg/agent/openflow/pipeline.go

```diff
@@ -1775,8 +1872,11 @@ func (f *featureNetworkPolicy) conjunctionActionPassFlow(conjunctionID uint32, t
 		conjReg = TFEgressConjIDField
 		nextTable = EgressRuleTable
 	}
-	flowBuilder := table.BuildFlow(ofPriority).MatchConjID(conjunctionID).
+	flowBuilder := table.BuildFlow(ofPriority)
+
+	flowBuilder = flowBuilder.MatchConjID(conjunctionID).
```

Comment:
No need to use two lines for flowBuilder

Reply:
done